src/battle_anim_new.c

```diff
@@ -3649,6 +3649,17 @@ const struct SpriteTemplate gDreepyMissileTemplate =
     .callback = AnimShadowBall
 };
 
+const struct SpriteTemplate gMouseCursorTemplate =
+{
+    .tileTag = ANIM_TAG_MOUSE_CURSOR,
+    .paletteTag = ANIM_TAG_MOUSE_CURSOR,
+    .oam = &gOamData_AffineOff_ObjNormal_16x32,
+    .anims = gDummySpriteAnimTable,
+    .images = NULL,
+    .affineAnims = gDummySpriteAffineAnimTable,
+    .callback = AnimHitSplatBasic
+};
+
```

Comment:
This is fully added with a working palette and everything but seems to be broken due to some kind of ass glitch so doesn't actually get used in the game